verifier/src/lib.rs

```diff
@@ -14,16 +14,20 @@ use miden_core::crypto::{
 };
 // EXPORTS
 // ================================================================================================
-pub use miden_core::{Kernel, ProgramInfo, StackInputs, StackOutputs, Word};
+pub use miden_core::{
+    Kernel, ProgramInfo, StackInputs, StackOutputs, Word,
+    precompile::{PrecompileError, PrecompileVerificationError, PrecompileVerifiers},
+};
 pub use winter_verifier::{AcceptableOptions, VerifierError};
 use winter_verifier::{crypto::MerkleTree, verify as verify_proof};
 pub mod math {
     pub use miden_core::{Felt, FieldElement, StarkField};
 }
 pub use miden_air::ExecutionProof;
-
+use miden_core::precompile::PrecompileRequest;
 // VERIFIER
```

Comment:
nit: I'd add a newline above this line.

Reply:
Fixed using a different syntax. Let me know if this works. If so, I think this might be worthwhile to apply across the codebase to make it clearer what we're exporting

```rust
mod exports {
  use crate::Type;
}
pub use exports::*;
```

Reply:
Ah - interesting! Yes, I think the new syntax works.